_test/test_mpi/finish_task_tester.m

```diff
@@ -63,6 +69,12 @@
     % if je.labIndex ~= 1
```

Comment:
commented out code block should be removed

Reply:
probably can be removed, though I suspect appeared due some problem with synchronization